src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import {
+  Footer as USWDSFooter,
```

Comment:
oh i have to remember this -- i have some naming collisions with search and this would solve it